cpp/src/parquet/arrow/arrow_reader_writer_test.cc

```diff
@@ -1387,8 +1394,8 @@ using TestLargeBinaryParquetIO = TestParquetIO<::arrow::LargeBinaryType>;
 TEST_F(TestLargeBinaryParquetIO, Basics) {
   const char* json = "[\"foo\", \"\", null, \"\xff\"]";
 
-  const auto large_type = ::arrow::large_binary();
-  const auto narrow_type = ::arrow::binary();
+  const auto large_type = ::arrow::large_utf8();
+  const auto narrow_type = ::arrow::utf8();
   const auto large_array = ::arrow::ArrayFromJSON(large_type, json);
```

Comment:
Why the change?

Reply:
I suggest this change because in Parquet this is defined as "valid utf8" . But you can decide that?

https://github.com/apache/parquet-format/blob/master/src/main/thrift/parquet.thrift#L154

Reply:
Just a mistake. Reverting.